roles/nf-core/defaults/main.yml

```diff
@@ -1,6 +1,6 @@
 nf_core_env_name: nf-core-env
 nf_core_env: "{{ sw_path }}/anaconda/envs/{{ nf_core_env_name }}"
-nf_core_version: 2.14.1
+nf_core_version: 3.0.2
```

Comment:
If you are upgrading the major version anyway, why only to 3.0.2 and not to 3.1.1?

Reply:
The playbook is configured to fetch nf-core from bioconda, but 3.1.1 is not available there yet :/  (see https://bioconda.github.io/recipes/nf-core/README.html)

Reply:
I see. I was asking because `nf-core pipelines download` has received relevant updates between the two versions with regard to Seqera Containers support. As long as the downloads work, there is no need to do something about it, but should there be an issue with Seqera Containers, it could be due to the outdated version.